Tests/ContributeTests/Spies/FrontMatterExporterSpies.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+import Contribute
+
+internal final class FrontMatterExporterSuccessfulSpy: FrontMatterExporter {
```

Comment:
I would create one Class and have take a Result property for which result you want